lib/cnet/tcp/tcp_input.c

```diff
@@ -90,14 +92,14 @@ tcp_input_lookup(struct cne_node *node, pktmbuf_t *m, struct pcb_hd *hd)
     pcb = cnet_pcb_lookup(hd, &key, BEST_MATCH);
     if (likely(pcb)) {
         int rc = TCP_INPUT_NEXT_PKT_DROP;
-
+#if 0        // Disable till cne_ipv4_udptcp_cksum_verify is fixed
         if (is_pcb_dom_inet6(pcb))
             csum = cne_ipv6_udptcp_cksum_verify(&tip->ip6, &tip->tcp);
         else
             csum = cne_ipv4_udptcp_cksum_verify(&tip->ip4, &tip->tcp);
```

Comment:
[cne_ipv4_udptcp_cksum_verify](https://github.com/CloudNativeDataPlane/cndp/blob/5f0a018298e518aeb24940d5237731c532bf0013/lib/include/net/cne_ip.h#L340C2-L349C2) invokes: __ipv4_udptcp_cksum() which expects the udp checksum to be 0

Overall this function doesn't look right to me. I would've assumed that it would compare the incoming UDP checksum with a calculated value... @KeithWiles what do you think?

```
static inline int
cne_ipv4_udptcp_cksum_verify(const struct cne_ipv4_hdr *ipv4_hdr, const void *l4_hdr)
{
    uint16_t cksum = __ipv4_udptcp_cksum(ipv4_hdr, l4_hdr);

    if (cksum != 0xffff)
        return -1;

    return 0;
}
```

Reply:
I thought when verifying the checksum if you run the checksum with the valid checksum in the header the result would be zero meaning the verify passed. Is that not the case?

Reply:
> I thought when verifying the checksum if you run the checksum with the valid checksum in the header the result would be zero meaning the verify passed. Is that not the case?

It does not seem to be....

Reply:
would need to do a local test to verify the valid checksum case and try to dig down into what the issue...

For now - incoming packets from the quicly client seem to be failing...

Reply:
So the issue was related to the layer 3 pointers we were using rather than the direct mbuf...